holoviews/operation/stats.py

```diff
@@ -100,8 +101,13 @@ def _process(self, element, key=None):
         elif bin_range[0] == bin_range[1]:
             bin_range = (bin_range[0]-0.5, bin_range[1]+0.5)
 
+        element_type = Area if self.p.filled else Curve
         data = data[np.isfinite(data)] if len(data) else []
         if len(data) > 1:
+            try:
+                kde = stats.gaussian_kde(data)
+            except LinAlgError:
```

Comment:
Are flat and empty distributions the only things that would cause a ``LinAlgError``?

Reply:
Only flat distributions as far as I know (as they result in a singular matrix), empty distributions are handled by the conditional on the line above.

Reply:
Actually, now I'm thinking that the empty case handled in this PR is just for violin plots. But maybe there are still other things that cause this exception?

Reply:
As I said I'm not aware of any other error conditions that would result in a linear algebra error, invalid datatypes would cause other errors and a singular matrix is the only error condition I'm aware of that could occur here.

Reply:
Github didn't show me your second comment before I posted my second one. Seems fine then.